kubemarine/procedures/check_paas.py

```diff
@@ -578,11 +578,11 @@ def kubernetes_dashboard_status(cluster: KubernetesCluster) -> None:
                         raise TestFailure("not available",hint=f"Please verify the following Kubernetes Dashboard status and fix this issue")
                 found_url = result.stdout
                 if ipaddress.ip_address(found_url).version == 4:
-                    check_url = cluster.nodes['control-plane'].get_first_member().sudo(f'curl -k -I https://{found_url}:443', warn=True)
+                    check_url = cluster.nodes['control-plane'].get_first_member().sudo(f'curl -k -I https://{found_url}:443 -s -o /dev/null -w "%{{http_code}}"', warn=True)
```

Comment:
Maybe logging of some extra information returned by curl can help the user understand what went wrong. At least, I suggest to add `-S`, but what about headers, do you think they are always useless for troubleshooting?

Reply:
I don't know how to use HTTP header in the kubernetes-dashboard troubleshooting.

Reply:
ok, leaving  as is.